src/App.jsx

```diff
@@ -1,5 +1,10 @@
+import { useState } from 'react';
+import cn from 'classnames';
+
 import 'bulma/css/bulma.css';
 import './App.scss';
+import { prependListener } from 'process';
```

Comment:
<img width="788" alt="image" src="https://github.com/mate-academy/react_list-of-goods-js/assets/70220667/7e5905f0-c36e-4d42-a622-29b5b9955a3e">